internal/services/netapp/netapp_volume_resource.go

```diff
@@ -398,6 +398,33 @@ func resourceNetAppVolume() *pluginsdk.Resource {
 				Default:     false,
 				Description: "Indicates whether the volume is a large volume.",
 			},
+
+			"cool_access": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"cool_access_retrieval_policy": {
+							Type:         pluginsdk.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringInSlice(volumes.PossibleValuesForCoolAccessRetrievalPolicy(), false),
+						},
+
+						"cool_access_tiering_policy": {
```

Comment:
since this is part of a `cool_access` block, we should drop this prefix from the contained attributes?
```suggestion
						"retrieval_policy": {
							Type:         pluginsdk.TypeString,
							Required:     true,
							ValidateFunc: validation.StringInSlice(volumes.PossibleValuesForCoolAccessRetrievalPolicy(), false),
						},

						"tiering_policy": {
```